sdk/python/kfp/v2/dsl/__init__.py

```diff
@@ -42,3 +42,11 @@
     ExitHandler,
     ParallelFor,
 )
+
+PIPELINE_JOB_NAME_PLACEHOLDER = '{{$.pipeline_job_name}}'
+PIPELINE_JOB_RESOURCE_NAME_PLACEHOLDER = '{{$.pipeline_job_resource_name}}'
+PIPELINE_JOB_ID_PLACEHOLDER = '{{$.pipeline_job_uuid}}'
+# RUN_ID_PLACEHOLDER is deprecated.
+RUN_ID_PLACEHOLDER = '{{$.pipeline_job_uuid}}'
```

Comment:
What is the plan for `RUN_ID_PLACEHOLDER`? If we want to deprecate it in v2, we shouldn't add it here.